### PR TITLE
Add missing shortcuts to documentation (Spoiler, Change Status)

### PR DESCRIPTION
**Pull request by makisukurisu**
Do we really need classes for tr?

### CODE REVIEW

File: Documentation/Shortcuts.md [hunk 1]
```diff
@@ -82,84 +82,88 @@
             <td class="column2 style1 s">Ctrl + Shift + M</td>
           </tr>
           <tr class="row19">
+            <td class="column1 style1 s">Spoiler</td>
+            <td class="column2 style1 s">Ctrol + Shift + P</td>
```
Comment (FrayxRulez) [inline Documentation/Shortcuts.md:86]: Typo:
```suggestion
            <td class="column2 style1 s">Ctrl + Shift + P</td>
```

**Review by FrayxRulez — CHANGES_REQUESTED** [review]
Can you also add change status shortcut? It's `Ctrl+Shift+Y`

**Review by FrayxRulez — APPROVED** [review]
Is it actually really needed to assign a different class (that I guess doesn't even exist) to each row?

**Comment by makisukurisu** [comment]
Classes are not necessary at all :sweat_smile: , wonder why they were there in the first place.

**Comment by FrayxRulez** [comment]
> Classes are not necessary at all 😅 , wonder why they were there in the first place.

So maybe you could remove them

**Comment by makisukurisu** [comment]
Done. I see no problems, but I'd like to ask someone to try to read this table with NVDA/Narrator just to make sure it's still accessible (because who knows - perhaps those classes were there for a reason).

Direct link to updated file:
https://github.com/UnigramDev/Unigram/blob/ab7d4c7323063c865a3369aced9ae6dba14989e7/Documentation/Shortcuts.md

**Comment by rehakp** [comment]
@makisukurisu Thanks for the update. Well, basically the table is accessible but it could be better by just rendering column headings by using HTML <th></th> elements instead of <td></td>. This way screen readers will be able to recognize them as such and announce them accordingly if configured by a user.

**Comment by makisukurisu** [comment]
@FrayxRulez , can you merge this?

**Comment by FrayxRulez** [comment]
@makisukurisu You can, it’s your pull request

**Review by MarcAnt01 — APPROVED** [review]

**Pull request merged**